packages/scikit-image/patches/preferred_plugins.patch

```diff
@@ -1,12 +0,0 @@
---- a/skimage/io/manage_plugins.py  2019-03-19 07:17:05.000000000 +0100
-+++ b/skimage/io/manage_plugins.py	2020-12-16 12:03:52.000000000 +0100
-@@ -38,7 +38,7 @@ plugin_meta_data = {}
- # the following preferences.
- preferred_plugins = {
-     # Default plugins for all types (overridden by specific types below).
--    'all': ['imageio', 'pil', 'matplotlib', 'qt'],
-+    'all': ['imageio', 'pil', 'matplotlib'],
```

Comment:
Did they change the default plugins, or is this now auto-detected my meson?

Reply:
The patch is not necessary any more due to a scikit-image change:
https://github.com/scikit-image/scikit-image/blob/f38bc6e511cb0605dcae4c31a17ba27f0de82b33/skimage/io/manage_plugins.py#L39-L44